src/backend/looped/vehicle/vehicle_ammo_special_type.cpp

```diff
@@ -0,0 +1,248 @@
+#include "backend/looped_command.hpp"
+
+namespace big
+{
+	class custom_vehicle_weapon : looped_command
+	{
+		using looped_command::looped_command;
```

Comment:
Is this really necessary? Why are you telling the compiler you're using the namespace looped_command when you're a child class of looped_command?

Reply:
It's not like I left 100 debug functions in there, it's 1 line. Maybe I can make changes in batch if more issues arise.

Reply:
@gir489returns all command implementations do this.